adal/authentication_context.py

```diff
@@ -75,11 +75,19 @@ def __init__(
             read timeout) <timeouts>` tuple.
         :param enable_pii: (optional) Unless this is set to True,
             there will be no Personally Identifiable Information (PII) written in log.
+        :param verify_ssl: (optional) requests verify. Either a boolean, in which case it 
+            controls whether we verify the server's TLS certificate, or a string, in which 
+            case it must be a path to a CA bundle to use. This value overrides env 
+            variable ADAL_PYTHON_SSL_NO_VERIFY.

```

Comment:
Suggest to change "This value overrides env variable ADAL_..." to "If this value is None, we will use the presence of env variable ADAL_PYTHON_SSL_NO_VERIFY to turn off SSL verification."

You can help to proofread it. The goal is to also explicitly document the usage of ADAL_PYTHON_SSL_NO_VERIFY as using its presence (or not) as a boolean; and we are NOT using ADAL_PYTHON_SSL_NO_VERIFY as a string path to a CA bundle.